packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -22,7 +22,10 @@ module.exports = async function(config) {
 		template = inProjectTemplatePath;
 	}
 
-	template = config.template || template;
+	if (config.template) {
+		const templatePathFromArg = resolve(cwd, config.template);
+		if (existsSync(templatePathFromArg)) template = templatePathFromArg;
```

Comment:
I feel like if someone specified a custom template path and it doesn't exist we should at least warn